Packs/MicrosoftDefenderAdvancedThreatProtection/Integrations/MicrosoftDefenderAdvancedThreatProtection/README.md

```diff
@@ -3776,3 +3998,435 @@ Creates a new indicator.
+                    {
+                        "ipAddress": "fe80::a998:1c4a:7e1c:4865",
+                        "macAddress": "123456789123",
+                        "operationalStatus": "Up",
+                        "type": "Ethernet"
+                    },
+                    {
+                        "ipAddress": "127.0.0.1",
+                        "macAddress": "",
+                        "operationalStatus": "Up",
+                        "type": "SoftwareLoopback"
+                    },
+                    {
+                        "ipAddress": "::1",
+                        "macAddress": "",
+                        "operationalStatus": "Up",
+                        "type": "SoftwareLoopback"
+                    },
+                    {
+                        "ipAddress": "fe80::5efe:1.2.3.4",
+                        "macAddress": "00000000000000E0",
+                        "operationalStatus": "Down",
+                        "type": "Tunnel"
+                    },
+                    {
+                        "ipAddress": "127.0.0.1",
+                        "macAddress": "00000000000000E0",
+                        "operationalStatus": "Up",
+                        "type": "Tunnel"
+                    },
+                    {
+                        "ipAddress": "fe80::2412:1420:53e0:f88b",
+                        "macAddress": "00000000000000E0",
+                        "operationalStatus": "Up",
+                        "type": "Tunnel"
+                    }
+                ],
+                "IsAADJoined": false,
+                "LastExternalIPAddress": "127.0.0.1",
+                "LastIPAddress": "1.2.3.4",
+                "LastSeen": "2022-01-26T22:21:19.2024139Z",
+                "MACAddress": "123456789123",
+                "OSBuild": 14393,
+                "OSPlatform": "WindowsServer2016",
+                "OSProcessor": "x64",
+                "OSVersion": "1607",
+                "RBACGroupID": 0,
+                "RiskScore": "None"
+            }
+        ]
+    }
+}
+```
+
+#### Human Readable Output
+
+>### Microsoft Defender ATP Machine:
+>|ID|ComputerDNSName|OSPlatform|LastIPAddress|LastExternalIPAddress|HealthStatus|RiskScore|ExposureLevel|
+>|---|---|---|---|---|---|---|---|
+>| f3bba49a | ec2amaz-ua9hieu | WindowsServer2016 | 1.2.3.4 | 127.0.0.1 | Active | None | High |
+
+
+### microsoft-atp-indicator-batch-update
+***
+Updates batch of indicator. If an indicator does not exist, a new indicator will be created.
+
+##### Required Permissions
+Ti.ReadWrite
+Ti.ReadWrite.All
+
+##### Limitations
+1. Rate limitations for this API are 30 calls per minute.
+2. There is a limit of 15,000 active indicators per tenant.
+3. Maximum batch size for one API call is 500.
+
+##### Note
+Please read [here](https://docs.microsoft.com/en-us/microsoft-365/security/defender-endpoint/ti-indicator?view=o365-worldwide) about a Microsoft Defender for Endpoint indicator resource type
```

Comment:
@bakatzir FYI (related to https://github.com/demisto/content/pull/17315/files#r797346784)